@here/harp-mapview/lib/Utils.ts

```diff
@@ -190,8 +194,11 @@ export namespace MapViewUtils {
         rayCaster.set(cameraZPosition, pointInCameraSpace.normalize());
 
         const vec3 = new THREE.Vector3();
+        if (elevation !== undefined) {
+            groundPlane.constant = -elevation;
```

Comment:
Would it be safer in this case just to create a new Vector3 for the groundPlane?
Setting and then clearing the global could cause undefined behavior if this function is ever called simultaneously.

Reply:
My understanding is that because of the event loop, we can only execute one part of our code at once, so the function can't be shared.

In the case of Web Workers, then they have their own copy, so nothing is shared. @zbigg, @robertoraggi, @StefanDachwitz  is that correct?

Reply:
Since rayCaster.ray.intersectPlane is not async so this should be fine.